versioned_sidebars/version-1.3.0-sidebars.json

```diff
@@ -208,6 +208,22 @@
         "add-ons/add-ons-slack-metrics"
         ]
     },
+    {
+      "type": "category",
+      "label": "Use Cases",
+      "collapsed": true,
+      "link": {
+        "type": "generated-index",
+        "title": "Add-ons",
+        "description": "The use case section provides details the usecases applicable to kubeslice",
```

Comment:
This section  (use case is repeated twice in a short sentence) 

details about 

kubeslice = KubeSlice.

End the sentence with a period/full stop after KubeSlice

Reply:
As this is the code, adding a period after KubeSlice gives me an error. kept as it is.